x-pack/plugins/security_solution/common/utils/risk_score_modules.test.ts

```diff
@@ -0,0 +1,312 @@
+  });
+});
+
+describe('getRiskScoreReduceScriptId', () => {
+  test('getRiskScoreReduceScriptId - host', () => {
+    const index = getRiskScoreReduceScriptId(RiskScoreEntity.host);
+    expect(index).toMatchInlineSnapshot(`"ml_hostriskscore_reduce_script_default"`);
+  });
+
+  test('getRiskScoreReduceScriptId - user', () => {
+    const index = getRiskScoreReduceScriptId(RiskScoreEntity.user);
+    expect(index).toMatchInlineSnapshot(`"ml_userriskscore_reduce_script_default"`);
+  });
+});
+
+describe('getLegacyIngestPipelineName', () => {
+  test('getLegacyIngestPipelineName - host', () => {
+    const name = getLegacyIngestPipelineName(RiskScoreEntity.host);
+    expect(name).toMatchInlineSnapshot(`"ml_hostriskscore_ingest_pipeline"`);
+  });
+
+  test('getLegacyIngestPipelineName - user', () => {
+    const name = getLegacyIngestPipelineName(RiskScoreEntity.user);
+    expect(name).toMatchInlineSnapshot(`"ml_userriskscore_ingest_pipeline"`);
+  });
+});
+
+describe('getLegacyRiskScoreLevelScriptId', () => {
+  test('getLegacyRiskScoreLevelScriptId - host', () => {
+    const index = getLegacyRiskScoreLevelScriptId(RiskScoreEntity.host);
+    expect(index).toMatchInlineSnapshot(`"ml_hostriskscore_levels_script"`);
+  });
+
+  test('getLegacyRiskScoreLevelScriptId - user', () => {
+    const index = getLegacyRiskScoreLevelScriptId(RiskScoreEntity.user);
+    expect(index).toMatchInlineSnapshot(`"ml_userriskscore_levels_script"`);
+  });
+});
+
+describe('getLegacyRiskScoreInitScriptId', () => {
+  test('getLegacyRiskScoreInitScriptId - host', () => {
+    const index = getLegacyRiskScoreInitScriptId(RiskScoreEntity.host);
+    expect(index).toMatchInlineSnapshot(`"ml_hostriskscore_init_script"`);
+  });
+
+  test('getLegacyRiskScoreInitScriptId - user', () => {
+    const index = getLegacyRiskScoreInitScriptId(RiskScoreEntity.user);
+    expect(index).toMatchInlineSnapshot(`"ml_userriskscore_init_script"`);
+  });
+});
+
+describe('getLegacyRiskScoreMapScriptId', () => {
+  test('getLegacyRiskScoreMapScriptId - host', () => {
+    const index = getLegacyRiskScoreMapScriptId(RiskScoreEntity.host);
+    expect(index).toMatchInlineSnapshot(`"ml_hostriskscore_map_script"`);
+  });
+
+  test('getLegacyRiskScoreMapScriptId - user', () => {
+    const index = getLegacyRiskScoreMapScriptId(RiskScoreEntity.user);
+    expect(index).toMatchInlineSnapshot(`"ml_userriskscore_map_script"`);
+  });
+});
+
+describe('getLegacyRiskScoreReduceScriptId', () => {
+  test('getLegacyRiskScoreReduceScriptId - host', () => {
+    const index = getLegacyRiskScoreReduceScriptId(RiskScoreEntity.host);
+    expect(index).toMatchInlineSnapshot(`"ml_hostriskscore_reduce_script"`);
+  });
+
+  test('getLegacyRiskScoreReduceScriptId - user', () => {
+    const index = getLegacyRiskScoreReduceScriptId(RiskScoreEntity.user);
+    expect(index).toMatchInlineSnapshot(`"ml_userriskscore_reduce_script"`);
+  });
+});
+
+describe('getRiskHostCreateLevelScriptOptions', () => {
```

Comment:
fixed in the file i sent you, but these do not have `test` functions so are not actually running tests